projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -15,7 +15,7 @@
 
 const FEATURE_NAME                             = 'subscriptions';
 const BLOCK_NAME                               = 'jetpack/' . FEATURE_NAME;
-const META_NAME_FOR_POST_LEVEL_ACCESS_SETTINGS = 'jetpack_newsletter_access';
+const META_NAME_FOR_POST_LEVEL_ACCESS_SETTINGS = '_jetpack_newsletter_access';
```

Comment:
Maybe we should have this constant only stored in one file ( i.e `projects/plugins/jetpack/extensions/blocks/subscriptions/constants.js` )

Reply:
We store this meta name [here](https://github.com/Automattic/jetpack/pull/26417/commits/303327fabd4e2867a7bc9a5ae26e4d8ebbe90114#diff-6c4e5f6a52aca3eb00a4cc6c302da733d70d8d26b61d4ba9e4ec2a8141add6e3R40) as well

Reply:
Nice catch! Any idea why we are not reusing `META_NAME_FOR_POST_LEVEL_ACCESS_SETTINGS`?!

Reply:
> Maybe we should have this constant only stored in one file ( i.e projects/plugins/jetpack/extensions/blocks/subscriptions/constants.js ). See [612f6a6](https://github.com/Automattic/jetpack/pull/26417/commits/612f6a632771636ba10a9feceb70b97a9ec8544d)

I'll take care of that.

Reply:
Fixed in [98f00f3](https://github.com/Automattic/jetpack/pull/26417/commits/98f00f36931a02f7ed6da7bbf1e63e8b60be33df)